docs/people/dev.md

```diff
@@ -3,22 +3,74 @@
 !!! success "Apply"
     Apply to this role [**here**](https://flexpa.applytojobs.ca/)
 
+Flexpa connects applications to health insurance data. We are looking to grow the team with a software developer who is excited to work on our mission of refactoring healthcare.
 
-At[ Flexpa](https://sandbox.flexpa.com), we're making it easy for patients to share their claims, benefit coverage, medications, and other data from health plans. We're a developer-first platform with rich APIs, robust developer documentation, and other tools to help applications to build novel patient and provider experiences powered by claims data.
+We've built a developer-first platform with rich APIs, hand-curated documentation, and other tools to help applications to build novel patient and provider experiences powered by claims data.
 
-If you join us in 2023 you will be an early team member and help us shape decisions about what we are building and why, who we hire, and company culture.
+We believe in the power of open standards and open culture. We surround ourselves with excellent people who want to do the best work of their careers. We expect this will be a good role for developers who share our values and want to work on hard problems.
```

Comment:
```suggestion
We believe in the power of open standards and open culture. We surround ourselves with excellent people who want to do the best work of their careers. We expect this will be a good role for developers who share our values and want to work on hard problems at a time where we accelerate alongside our initial cohort of customers.
```

Reply:
Gives some sense of the pace, as well as the opportunity of this particular moment in Flexpa.